src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -21,6 +21,7 @@
         </field>
         <field name="storageOptions" type="text" column="storageOptions" nullable="true" />
         <field name="mimeType" type="string" column="mimeType" length="255" nullable="true" />
+        <field name="properties" type="string" column="properties" length="1000" nullable="true" />
```

Comment:
1. Media Entity does not have 'properties' in XML.
2. Yes
3. Videos: saving duration;  could also be used for e.g. PDF -> amount of pages, etc.